Dockerfile

```diff
@@ -19,6 +19,8 @@ RUN GO111MODULE=on go mod download
 
 COPY . .
 
+RUN apk update && apk add bash
```

Comment:
seems below change not requiring `bash`

Reply:
install `bash` cause docker build failed on OS X